standard/standard_defines.php

```diff
@@ -911,165 +910,130 @@
+ * Image type constant used by the {@link image_type_to_mime_type()} and {@link image_type_to_extension()} functions.
  * @link https://php.net/manual/en/image.constants.php
  */
 define ('IMAGETYPE_JPX', 11);
 
 /**
- * Image type constant used by the
- * image_type_to_mime_type and
- * image_type_to_extension functions.
+ * Image type constant used by the {@link image_type_to_mime_type()} and {@link image_type_to_extension()} functions.
  * @link https://php.net/manual/en/image.constants.php
  */
 define ('IMAGETYPE_JB2', 12);
 
 /**
- * Image type constant used by the
- * image_type_to_mime_type and
- * image_type_to_extension functions.
+ * Image type constant used by the {@link image_type_to_mime_type()} and {@link image_type_to_extension()} functions.
  * @link https://php.net/manual/en/image.constants.php
  */
 define ('IMAGETYPE_SWC', 13);
 
 /**
- * Image type constant used by the
- * image_type_to_mime_type and
- * image_type_to_extension functions.
+ * Image type constant used by the {@link image_type_to_mime_type()} and {@link image_type_to_extension()} functions.
  * @link https://php.net/manual/en/image.constants.php
  */
 define ('IMAGETYPE_IFF', 14);
 
 /**
- * Image type constant used by the
- * image_type_to_mime_type and
- * image_type_to_extension functions.
+ * Image type constant used by the {@link image_type_to_mime_type()} and {@link image_type_to_extension()} functions.
  * @link https://php.net/manual/en/image.constants.php
  */
 define ('IMAGETYPE_WBMP', 15);
 
 /**
- * Image type constant used by the
- * image_type_to_mime_type and
- * image_type_to_extension functions.
+ * Image type constant used by the {@link image_type_to_mime_type()} and {@link image_type_to_extension()} functions.
  * @link https://php.net/manual/en/image.constants.php
  */
 define ('IMAGETYPE_JPEG2000', 9);
 
 /**
- * Image type constant used by the
- * image_type_to_mime_type and
- * image_type_to_extension functions.
+ * Image type constant used by the {@link image_type_to_mime_type()} and {@link image_type_to_extension()} functions.
  * @link https://php.net/manual/en/image.constants.php
  */
 define ('IMAGETYPE_XBM', 16);
 
 /**
- * Image type constant used by the
- * image_type_to_mime_type and
- * image_type_to_extension functions.
- * (Available as of PHP 5.3.0)
+ * Image type constant used by the {@link image_type_to_mime_type()} and {@link image_type_to_extension()} functions.
  * @link https://php.net/manual/en/image.constants.php
+ * @since 5.3.0
  */
 define ('IMAGETYPE_ICO', 17);
-define ('IMAGETYPE_UNKNOWN', 0);
-define ('IMAGETYPE_COUNT', 18);
 
 /**
- * Image type constant used by the image_type_to_mime_type() and image_type_to_extension() functions
+ * Image type constant used by the {@link image_type_to_mime_type()} and {@link image_type_to_extension()} functions.
  * @link https://php.net/manual/en/image.constants.php
+ * @since 7.1.0
  */
 define('IMAGETYPE_WEBP', 18);
+define('IMAGETYPE_UNKNOWN', 0);
```

Comment:
Why move those?

Reply:
Logical order: _actual image types_ -> _others (unknown/count)_

I can revert if it's a problem here.

Reply:
I guess it's better to revert (if the logical order is the only reason), because our stubs are not intended to be read by users:)

Reply:
I tell you from daily experience with PhpStorm:

My collegues (including myself from time to time) tend to Ctrl+Click on standard PHP identifiers to go to declaration instead of using Ctrl+Q for Quick Definition.
So we end up in stubs file looking at the "code" and keep looking around. It just helps a little if things are ordered somehow there to see "what else" is defined for example with `IMAGETYPE_`.

Reply:
I claim that more people are looking into stubs files then people are watching the git history of them on Github or wherever else you see it.

Reply:
Okay, I see. We'll take it into consideration then. Still, please, resolve my first remark and I'll merge your changes :)